### DISCUSSION TITLE
New tile give me `app/Http/Livewire/" directory does not exist`

### DISCUSSION THREAD

**Discussion by rachids** [spatie/laravel-dashboard#80] [Q&A] [answered]
Hello everyone,

So I installed the dashboard successfully, added a community tile without any trouble and now I want to make a custom one for my needs.

Since I couldn't manage to show the tile, I decided to strictly follows what's in the documentation.

So I created a `DummyComponent` inside `App\Tiles` and a view in `resources/views/tiles/dummy.blade.php`, but as soon as I add the following code, Laravel crashes because it can't find the Livewire directory:
```html
<livewire:dummy position="a2"/>
```

I scouted the spatie's dashboard repo and there isn't any Livewire directory in their `app/Http`.

I even published the livewire's config file and edited the `class_namespace` to point towards `app\Tiles`, without success.

Am I missing something ? 

Edit: ~~Is this because I'm on Windows currently? I find it odd that it uses the normal slashes in the error output.~~
Deployed the project to my raspberry pi and still have the issue 😢 

PS- here are the versions of my toolset:

```
PHP 8.0.0
Laravel Framework 8.53.1
spatie/laravel-dashboard * 2.0.1
```

**Comment by YoanG17** [comment, marked as answer]
> Hello everyone,
> 
> So I installed the dashboard successfully, added a community tile without any trouble and now I want to make a custom one for my needs.
> 
> Since I couldn't manage to show the tile, I decided to strictly follows what's in the documentation.
> 
> So I created a `DummyComponent` inside `App\Tiles` and a view in `resources/views/tiles/dummy.blade.php`, but as soon as I add the following code, Laravel crashes because it can't find the Livewire directory:
> 
> ```
> <livewire:dummy position="a2"/>
> ```
> 
> I scouted the spatie's dashboard repo and there isn't any Livewire directory in their `app/Http`.
> 
> I even published the livewire's config file and edited the `class_namespace` to point towards `app\Tiles`, without success.
> 
> Am I missing something ?
> 
> Edit: ~Is this because I'm on Windows currently? I find it odd that it uses the normal slashes in the error output.~ Deployed the project to my raspberry pi and still have the issue 😢
> 
> PS- here are the versions of my toolset:
> 
> ```
> PHP 8.0.0
> Laravel Framework 8.53.1
> spatie/laravel-dashboard * 2.0.1
> ```

Try to create folder with this command : 

php artisan make:livewire counter
php artisan livewire:discover

and retry to display your page.
